app/screens/channel_add_members/channel_add_members.js

```diff
@@ -33,7 +33,7 @@ export default class ChannelAddMembers extends PureComponent {
             handleAddChannelMembers: PropTypes.func.isRequired,
             searchProfiles: PropTypes.func.isRequired,
         }).isRequired,
-        currentChannelId: PropTypes.string.isRequired,
+        currentChannel: PropTypes.object.isRequired,
```

Comment:
If you need the group constraint property you can pass it individually as well to avoid unnecessary re-renders when other properties of the channel object changes.

Also make sure that the property exists as it may not be available in older servers.

Reply:
Thanks for spotting this! Fixed passing both properties and adding `defaultProps` to the component